### DISCUSSION TITLE
Search part of file - did not include the current folder

### DISCUSSION THREAD

**Discussion by docop** [sharkdp/fd#1404] [General]
Hi
 I would like to know why the main folder where we do the command is not included when we search for a file ?
  from home folder  make 1 file:  tst5.ss  And do: mkdir tst2   ,  mkdir tst4 ,    inside these folder do file :   xxxgstt5.jpg    and  sdft5.jpg
then in home folder do :
`fdfind -g *t5.jpg    `          # it will list file in those 2 sub folder  = 2 file found
`fdfind -g *gst*.jpg   `        # it will list file in those 2 sub folder = 2 file found
  BUT
`fdfind -g *t5.*     `         # 1 file found
 will return only the .ss   and not list the .jpg of the sub folder.   Is this a bug ?

 The easy equivalent is :`   dir *t5.* /s     `        # that will list :   .ss  and the 2 jpg file in the 2 sub folder.  so a result of 3 file found.

**Comment by tmccombs** [comment]
I think probably what is happening is that your shell is expanding "*t5.*" to "tst5.ss" before passing it to tfd. and then the only file that matches is the `tst5.ss` file in your home directory.

Try running `fdfind -g '*t5.*'`. Note the single quotes, that will prevent your shell from doing glob expansion.

**Comment by docop** [comment]
hum indeed it do work with the  ' ' . But is there a way to fix as a symlink or so to not have to add extra colon ..  ?  thanks again
